### ISSUE TITLE
Improper heading structure globally

### ISSUE THREAD

**Issue by avaaec** [ManifoldScholar/manifold#2261]
This is an issue that appears in multiple places around manifold, much like global issues around misuse of nav elements, groupings, and single item lists. Collectively, small but recurring issues like this have a significant negative impact on accessibility. I will focus on one specific instance of this issue.
## Steps To Reproduce
1. Go to reader
2. Open "Your Notes" Panel
3. Notice that there is a level three heading "SHOW YOUR:" with a level three heading under it "You haven't annotated this text yet"
4. Notice that there is no logical semantic connection or visual hierarchical relationship between these elements.
## Observed Behavior
It appears that in many places in manifold headings are being used to style the font without concern for semantics. This is a problem because many assistive technologies rely on heading hierarchy to communicate page structure and layout. For example I use a screen reader to quickly understand the layout of information on a page. Here is an example of how the current heading structure appears to me versus how it might appear if headings were used semantically.
### Bad
```
AStudyinScarletTitlepage
Your Notes
--->SHOW YOUR:
--->--->You haven't annotated this text yet
--->Imprint (The name of the first chapter)
```
### Good
```
A Study in Scarlet Title Page
--->Your Notes
--->---> You haven't annotated this text yet
--->Imprint (The name of the first chapter)
```
## Expected Behavior
Headings convey semantic information to assistive technologies and should be hierarchical, beginning with H1. Headings should never be used to style text, that's what CSS is for. Hierarchies in headings should make since logically based on visual relationship and informational hierarchy.
## Testing Environment
edge.manifoldapp.org
NVDA 2019.1.1
ESR Firefox 60.7.0esr

**Comment by dananjohnson** [comment]
Thank you for the feedback! We resolved this issue for the frontend and backend of the application in e35422d, but still need to tackle the reader.

**Comment by avaaec** [comment]
This fix is verified.
